code/datums/mutations/speech.dm

```diff
@@ -230,3 +230,55 @@
 	..()
 	owner.grant_language(/datum/language/common)
 	owner.remove_language(/datum/language/beachbum)
+
+
+	/datum/mutation/human/italian
+	name = "Italian"
+	quality = MINOR_NEGATIVE
+	dna_block = NON_SCANNABLE
+	text_gain_indication = "<span class='notice'>You feel-a the spicy meatballs and-a pasta singing to your-a soul!</span>"
+	text_lose_indication = "<span class='notice'>You no longer feel the urge to switch sides midway through a war. Your love of pasta passes.</span>"
+
+/datum/mutation/human/italian/say_mod(message)
+	if(message)
+		message = " [message] "
+		message = replacetext(message,"mom","mamma")
+		message = replacetext(message,"dad","pappa")
+		message = replacetext(message,"baby",pick("bambino","little sausage roll"))
+		message = replacetext(message,"spicy","a-spicy")
+		message = replacetext(message,"I'm","I'm-a")
+		message = replacetext(message,"friend","enemy")
+		message = replacetext(message,"enemy","friend")
+		message = replacetext(message,"traitor","mafioso")
+		message = replacetext(message,"operative","greek")
+		message = replacetext(message,"op","greek")
+		message = replacetext(message,"ops","greeks")
+		message = replacetext(message,"nuke","the spiciest-a meatball")
```

Comment:
Wouldn't "the nuke" filter to "the the spiciest-a meatball"?
Also my sides are in orbit, PR of the year